relentless/simulate/dilute.py

```diff
@@ -1,3 +1,18 @@
+"""
+Dilute system
+=============
+
+Provides the :meth:`AddEnzembleAnalyzer()` operation for a dilute system.
```

Comment:
This should describe what a dilute simulation is. Alternatively, put it in the `Dilute` class documentation and include that here. (That might be better.)